### ISSUE TITLE
Exception: "Access is allowed from event dispatch thread only."

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#177]
plugin version: 0.78.2

I've gotten 6 of these exceptions in the last 20 minutes, all from `om.reason.ide.docs.DocumentationProvider.getQuickNavigateInfo`

```
Access is allowed from event dispatch thread only.

com.intellij.openapi.diagnostic.RuntimeExceptionWithAttachments: EventQueue.isDispatchThread()=false Toolkit.getEventQueue()=com.intellij.ide.IdeEventQueue@20f8b349
Current thread: Thread[ApplicationImpl pooled thread 343,4,Idea Thread Group] 1379211519
SystemEventQueueThread: Thread[AWT-EventQueue-0 2019.2#IC-192.5728.98 IDEA, eap:false, os:Windows 10 10.0, java-version:JetBrains s.r.o 11.0.3+12-b304.10 2019.2#IC-192.5728.98 IDEA, eap:false, os:Windows 10 10.0, java-version:JetBrains s.r.o 11.0.3+12-b304.10,6,Idea Thread Group] 1402150453
	at com.intellij.openapi.application.impl.ApplicationImpl.assertIsDispatchThread(ApplicationImpl.java:1068)
	at com.intellij.openapi.application.impl.ApplicationImpl.assertIsDispatchThread(ApplicationImpl.java:1057)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.getSelectedEditorWithProvider(FileEditorManagerImpl.java:1332)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.getSelectedEditor(FileEditorManagerImpl.java:1324)
	at com.reason.ide.docs.DocumentationProvider.getQuickNavigateInfo(DocumentationProvider.java:121)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.getQuickNavigateInfo(CompositeDocumentationProvider.java:112)
	at com.intellij.codeInsight.navigation.CtrlMouseHandler.generateInfo(CtrlMouseHandler.java:282)
	at com.intellij.codeInsight.navigation.CtrlMouseHandler.access$1400(CtrlMouseHandler.java:91)
	at com.intellij.codeInsight.navigation.CtrlMouseHandler$InfoSingle.getInfo(CtrlMouseHandler.java:402)
	at com.intellij.codeInsight.navigation.CtrlMouseHandler$TooltipProvider.doExecute(CtrlMouseHandler.java:731)
	at com.intellij.codeInsight.navigation.CtrlMouseHandler$TooltipProvider.access$2100(CtrlMouseHandler.java:646)
	at com.intellij.codeInsight.navigation.CtrlMouseHandler$TooltipProvider$1.performInReadAction(CtrlMouseHandler.java:700)
	at com.intellij.openapi.progress.util.ReadTask.lambda$runBackgroundProcess$0(ReadTask.java:66)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:936)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:57)
	at com.intellij.openapi.progress.util.ReadTask.runBackgroundProcess(ReadTask.java:66)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runUnderProgress$4(ProgressIndicatorUtils.java:231)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$3(CoreProgressManager.java:185)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:169)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:156)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:185)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runUnderProgress(ProgressIndicatorUtils.java:229)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.access$100(ProgressIndicatorUtils.java:37)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils$4.run(ProgressIndicatorUtils.java:180)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by Coobaha** [comment]
the same issue https://github.com/reasonml-editor/reasonml-idea-plugin/issues/172

**Comment by giraud** [comment]
should be fixed in 0.78.3
